client/src/lib/sessions/hooks/useCompletedSessions.ts

```diff
@@ -1,13 +1,17 @@
 import {useMemo} from 'react';
 
-import useCurrentUserState from '../../user/hooks/useCurrentUserState';
+import useUserEvents from '../../user/hooks/useUserEvents';
 
 const useCompletedSessions = () => {
-  const userState = useCurrentUserState();
+  const {completedSessionEvents} = useUserEvents();
 
   const completedSessions = useMemo(
-    () => userState?.completedSessions ?? [],
-    [userState],
+    () =>
+      completedSessionEvents.sort(
+        (a, b) =>
+          new Date(a.timestamp).getTime() - new Date(b.timestamp).getTime(),
```

Comment:
Stupid question: does this mean that a being more recent will go to the top?

Cause on the journey use case, we would display the most recent ones in the end of the list's section

Reply:
No, the opposite, latest last